src/main/java/org/openstreetmap/atlas/geography/sharding/AppendCountsForAllZoom.java

```diff
@@ -0,0 +1,179 @@
+package org.openstreetmap.atlas.geography.sharding;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
+import org.openstreetmap.atlas.streaming.Streams;
+import org.openstreetmap.atlas.streaming.resource.File;
+import org.openstreetmap.atlas.streaming.resource.WritableResource;
+import org.openstreetmap.atlas.utilities.collections.StringList;
+import org.openstreetmap.atlas.utilities.runtime.Command;
+import org.openstreetmap.atlas.utilities.runtime.CommandMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Command that takes a file with feature counts for each {@link SlippyTile} at a certain zoom
+ * level, and then calculates summed counts for every lower zoom layer to the file. This is meant to
```

Comment:
Nit: the wording here is a little awkward with "every lower zoom layer to the file" - do you mind updating?